### ISSUE TITLE
Errors in Sporadic.tst in GAP stable branch

### ISSUE THREAD

**Issue by alex-konovalov** [gap-packages/recog#41]
Testing GAP 4.10.1 release candidate, when all packages are loaded I observe this in [this Jenkins test (St Andrews access only](https://gap-ci.cs.st-andrews.ac.uk/view/GAP-release/job/GAP-minor-release-test/GAPCOPTS=64build,GAPTARGET=packages,label=kovacs/129/consoleFull):
```
########> Diff in /circa/scratch/gap-jenkins/workspace/GAP-minor-release-test/\
GAPCOPTS/64build/GAPTARGET/packages/label/kovacs/gap-4.10.1/pkg/recog-1.3.1/ts\
t/quick/Sporadic.tst:47
# Input is:
TestSporadic("Co2");
# Expected output:
[ "Co2" ]
# But found:
false
########
```

**Comment by alex-konovalov** [comment]
Logged into Jenkins build workspace, called 
```
LoadAllPackages();
TestPackage("recog");
```
and got this:
```
########> Diff in /circa/scratch/gap-jenkins/workspace/GAP-pkg-update-stable-q\
uicktest/GAPCOPTS/64build/GAPGMP/gmp/GAPTARGET/packages/label/kovacs/GAP-pkg-u\
pdate-stable-snapshot/pkg/recog-1.3.1/tst/quick/Sporadic.tst:33
# Input is:
TestSporadic("M24");
# Expected output:
[ "M24" ]
# But found:
[ "Co3" ]
########
```

**Comment by alex-konovalov** [comment]
Managed also to get this:
```
########> Diff in /circa/scratch/gap-jenkins/workspace/GAP-pkg-update-stable-q\
uicktest/GAPCOPTS/64build/GAPGMP/gmp/GAPTARGET/packages/label/kovacs/GAP-pkg-u\
pdate-stable-snapshot/pkg/recog-1.3.1/tst/quick/Sporadic.tst:77
# Input is:
TestSporadic("Fi24'.2"); # FIXME: sometimes "recognized" as M
# Expected output:
[ "Fi24'.2" ]
# But found:
[ "M", "Fi24'.2" ]
########
```

**Comment by alex-konovalov** [comment]
Wondering now whether there is some other updated package which broke the test for recog package, or this is is something that may happen in recog, and was uncovered by changing test environment...

**Comment by fingolfin** [comment]
This is most likely due to the state of the random number generator. So this could then be fixed by adding `START_TEST` to the file, except I explicitly do not want this, because we need to be able to debug those tests with different RNG states. So another fix might be to reset the RNG state in `tst/testall.g`.

**Comment by alex-konovalov** [comment]
Ok, then this should have no implications on the collection of packages included with GAP 4.10.1 - good. 

Failing test could now be seen on Travis [here](https://travis-ci.org/gap-infra/gap-docker-pkg-tests-stable-4.10/jobs/497007589).

**Comment by fingolfin** [comment]
I've pushed a fix for this to master, but have not yet released it.
